serverless.yml

```diff
@@ -1,17 +1,35 @@
-service:
-  name: nversary
+service: nversary
+
+frameworkVersion: '2'
 
-# Add the serverless-webpack plugin
 plugins:
-  - serverless-webpack
+  - serverless-plugin-typescript
 
 provider:
   name: aws
-  runtime: nodejs12.x
+  runtime: nodejs14.x
   region: eu-west-1
+  lambdaHashingVersion: 20201221
+  iam:
+    role:
+      statements:
+        - Effect: 'Allow'
+          Action:
+            - 'ssm:GetParameter'
+          Resource:
+            Fn::Sub: "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter${env:SSM_PARAMETER_NAME}"
+        - Effect: 'Allow'
+          Action:
+            - 's3:GetObject'
+          Resource:
+            - "arn:aws:s3:::${env:PEOPLE_S3_BUCKET}/${env:PEOPLE_S3_KEY}"
 
 functions:
   nversaryGreeter:
     handler: handler.greeter
     events:
-    - schedule: cron(50 3 * * ? *)
\ No newline at end of file
+      - schedule: cron(50 3 * * ? *)
```

Comment:
Why at 3:50 during the night? Or is this some timezone thing? :D Would during the working day be nicer?

Reply:
It must run after people.json generator runs and before the first scheduled message sending time 7:50 UTC. Otherwise doen't matter much when it runs.